tests/runtests.py

```diff
@@ -247,6 +247,36 @@ def test_cmd_hexdump(self):
         self.assertNoException(res)
         return
 
+    def test_cmd_memory_watch(self):
```

Comment:
Would be nice if you wrote something to memory and then ensured that it showed up in the context.

Reply:
agreed. i will put it on my todo list ;)